jenkins_pipelines/environments/uyuni-mu-cloud

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env groovy
```

Comment:
Same here, I would prefer if we mention aws in the file.

Reply:
Was marked as resolved, but I still can't see aws in the filename, any reason to don't have it?